hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/audit/persistence/AbstractAuditPersistence.java

```diff
@@ -226,11 +227,11 @@ private String getServer() {
 	 * @param identifier
 	 * @param direction
 	 */
-	private void populateAffectedParty(AffectedParty affectedParty, UUID transactionUuid, String identifier, String direction) {
+	private void populateAffectedParty(AffectedParty affectedParty, UUID transactionUuid, String identifier, AffectedPartyDirection direction) {
 	    affectedParty.setTransactionId(transactionUuid);						
 		affectedParty.setIdentifier(identifier);	
 		affectedParty.setIdentifierType(BCPHN);
-		affectedParty.setDirection(direction);
+		affectedParty.setDirection(direction.getValue());
```

Comment:
If you use getValue() now then Inbound is being stored but spec is INBOUND. Can just do same as other enums for audits.

Reply:
I rechecked the spec and it is lowercase for these values so code is correct as is. Some other preset values like "Transaction Start" are lowercase also, just some types are uppercase.